lms/js_config_types.py

```diff
@@ -36,7 +36,12 @@ class APICourse(TypedDict):
 
 
 class APIStudent(TypedDict):
-    id: str
+    h_userid: str
+    """ID of the student in H."""
+
+    lms_id: str
```

Comment:
Deviating here from the name in the model (user_id). lms_id is IMO a better name, we use that in some other tables (eg. grouping).